File: src/codes/Users_Manual/modulation/fourier_2.cs
~using AUTD3Sharp;
~using AUTD3Sharp.Modulation;
~using static AUTD3Sharp.Units;
~using System;
var m = new Fourier([new Sine(100u * Hz), new Sine(150 * Hz).WithPhase(MathF.PI / 2.0f * rad)]);

File: src/codes/Users_Manual/modulation/fourier_2.cpp
//~#include<autd3.hpp>
//~#include <autd3/modulation/fourier.hpp>
//~int main() {
const auto m = autd3::modulation::Fourier(
    {autd3::modulation::Sine(100 * autd3::Hz),
     autd3::modulation::Sine(150 * autd3::Hz)
         .with_phase(autd3::pi / 2.0f * autd3::rad)});
//~return 0; }